BurnerB/Questioner-ADC-
# Bg fix question endpoints  163111052

#### What does this PR do?
FIx question endpoint
#### Description of Task to be completed?
Fix question endpoint ,should respond with the specified data in challenge 2

#### How should this be manually tested?
clone this repo
CD into it
Set up a virtual environment by:
`sudo install virtualenv env`
then `source venv/bin/activate`

install all modules in requirements.txt
run flask by:
`FLASK_APP=run.py`
`FLASK_ENV =development`
`Flask run`
The app should now be running
copy the  routes :
`POST /questions`

Paste the route in postman and test 

#### What are the relevant pivotal tracker stories?
#163111052
#### Screenshots (if appropriate)
![create question](https://user-images.githubusercontent.com/45785786/51076625-d6c58b80-16ab-11e9-8514-ba5706660535.png)
